code/style.css

```diff
@@ -1,7 +1,398 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body,
+html {
+  margin: 0px;
+  font-family: "open sans", sans-serif;
+  background: linear-gradient(180deg, rgb(34,36,25) 46%, rgb(77,72,43) 100%);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/*** HERO ***/
+
+.hero-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  object-fit: cover;
+  position: relative;
+  overflow: hidden;
+  color: #ECC8AF;
+}
+
+.hero-content {
+  position: absolute;
+  width: 100%;
+  text-align: center;
+  margin-bottom: 45px;
+}
+
+.hero-headline {
+  margin: 0 auto 20px;
+  max-width: 650px;
+}
+
+.hero-info {
+  opacity: 0.8;
+  margin: 0 auto 20px;
+  max-width: 650px;
+}
+
+.hero-action {
+  background-color: rgba(255,255,255,0.2);
+  color: #ECC8AF;
+  text-align: center;
+  backdrop-filter: blur(15px);
+  margin-top: 0.3em;
+  font-size: 20px;
+  transition: all 0.15s ease-in-out;
+  transform: scale(1);
+  white-space: nowrap;
+  height: 40px;
+  line-height: 40px;
+  border-radius: 20px;
+  display: inline-block;
+  text-decoration: none;
+  width: 200px;
+  padding: 0 20px;
+}
+
+.hero-action:hover {
+  transform: scale(1.03);  
+  background-color: rgba(255,255,255,0.3);
+}
+
+.hero-container img {
+  width: 100%;
+}
+
+/*** MAIN ***/
+
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.form-container, 
+.accordion-container {
+  padding: 20px;
+  width: 100%;
+}
+
+.section-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+.main-header {
+  margin: 0;
+  padding: 20px 0;
+  text-align: center;
+}
+
+/*** FORM ***/
+
+.form-container {
+  background: #ECC8AF;
+  color: #3E4E50;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+}
+
+.form-label {
+  width: 100%;
+  padding: 10px 0;
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.form-input {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.form-input::placeholder {
+  font-size: 14px;
+}
+
+.form-input:valid {
+  border-bottom: 1.5px solid green;
+}
+
+.form-input[type="text"],
+.form-input[type="email"],
+.form-input[type="password"] {
+  border: none;
+  border-bottom: 1px solid rgb(77,72,43);
+  background: rgba(89,89,75, 0.2);
+  outline: none;
+  height: 40px;
+  font-size: 16px;
+}
```

Comment:
You've done some amazing styling for your inputs but it's bad practise to remove the outline due to accessibility. E.g. if someone is using a screen reader or other visual aids. Perhaps you could change the color instead?